test/functional/wallet_bumpfee.py

```diff
@@ -231,13 +231,14 @@ def test_unconfirmed_not_spendable(rbf_node, rbf_node_address):
     assert_equal([t for t in rbf_node.listunspent(minconf=0, include_unsafe=False) if t["txid"] == bumpid], [])
 
     # submit a block with the rbf tx to clear the bump tx out of the mempool,
-    # then call abandon to make sure the wallet doesn't attempt to resubmit the
-    # bump tx, then invalidate the block so the rbf tx will be put back in the
-    # mempool. this makes it possible to check whether the rbf tx outputs are
+    # then invalidate the block so the rbf tx will be put back in the mempool.
+    # Also, call abandon to make sure the wallet doesn't attempt to resubmit
```

Comment:
Isn't this comment misleading if the call to abandon doesn't actually do anything? Seems better to remove the comment and call entirely and maybe replace with a comment that says "this test assumes the wallet will not reinsert the bump transaction after the call to invalidateblock."